### ISSUE TITLE
Update text styles, colors, and spacing (redline)

### ISSUE THREAD

**Issue by ashin-czi** [chanzuckerberg/single-cell-data-portal#4207]
- [x] Filter dropdown labels should be 500 weight (Dataset, Disease, Self-Reported Ethnicity, Publication, Sex, Tissue)
- [x] Selected option in the dropdown closed state (organism, group by, sort cell types, sort genes, color scale) should also be 500 weight. [Image for clarity](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBM09MQVE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--a951a9a9258edceacc59c2e4620e854adda87112/image.png)
- [x] Label for sort cell types, sort genes, and color scale should all be `fontBodyXXS/500` in `Text Secondary` color.

**Comment by kaloster** [comment]
This ticket feels outdated: https://www.figma.com/file/JS5ThqTBcpAVuBFglAItKf/Gene-Expression-v1?type=design&node-id=4866-443488&mode=design&t=wK2DH5imz2a5J9b9-0
the UI elements have changed since and there's another ticket that addresses any inconsistencies (which was partially completed already) - Gene Expression consistency quick fixes #5416

As for hardcoded styles - for the most part it has dynamic values and whenever needed it is addressed as part of the refactor epic.

So in summary: imo this can either be closed or updated with details to point out specific inconsistencies

cc: @signechambers1 @karenliang0

**Comment by signechambers1** [comment]
@karenliang0 can you take a look and update any inconsistencies? Thank you!

**Comment by karenliang0** [comment]
Thank you for looking into this, Ronen!
Found these inconsistencies regarding 500 weight font.

- [ ] Filter dropdown labels should be 500 weight (Dataset, Disease, Self-Reported Ethnicity, Publication, Sex, Tissue)
- [ ] Selected option in the dropdown closed state (organism, group by, sort cell types, sort genes, color scale) should also be 500 weight. [Private Zenhub Image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBM09MQVE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--a951a9a9258edceacc59c2e4620e854adda87112/image.png)
- [ ] Label for sort cell types, sort genes, and color scale should all be `fontBodyXXS/500` in `Text Secondary` color.

**Comment by signechambers1** [comment]
Updating the body of the ticket, thanks @karenliang0 !
